code/modules/admin/verbs/adminhelp.dm

```diff
@@ -141,3 +141,234 @@ var/list/adminhelp_ignored_words = list("unknown","the","a","an","of","monkey","
+					if(X.get_preference_value(/datum/client_preference/staff/play_adminhelp_ping))
+						X << 'sound/effects/adminhelp_new.ogg'
+					X << mentor_msg
+			if(modholders.len)
+				for(var/client/X in modholders) // Mods
+					if(X.get_preference_value(/datum/client_preference/staff/play_adminhelp_ping))
+						X << 'sound/effects/adminhelp_new.ogg'
+					X << msg
+			if(adminholders.len)
+				for(var/client/X in adminholders) // Admins get the full monty
+					if(X.get_preference_value(/datum/client_preference/staff/play_adminhelp_ping))
+						X << 'sound/effects/adminhelp_new.ogg'
+					X << msg
+		if("Rule Issue")
+			if(modholders.len)
+				for(var/client/X in modholders) // Mods
+					if(X.get_preference_value(/datum/client_preference/staff/play_adminhelp_ping))
+						X << 'sound/effects/adminhelp_new.ogg'
+					X << msg
+			if(adminholders.len)
+				for(var/client/X in adminholders) // Mods
+					if(X.get_preference_value(/datum/client_preference/staff/play_adminhelp_ping))
+						X << 'sound/effects/adminhelp_new.ogg'
+					X << msg
+
+
+
+
+/*	for(var/client/X in GLOB.admins)
+		if((R_ADMIN|R_MOD|R_MENTOR) & X.holder.rights)
+			if(X.is_afk())
+				admin_number_afk++
+			if(X.get_preference_value(/datum/client_preference/staff/play_adminhelp_ping) == GLOB.PREF_HEAR)
+				sound_to(X, 'sound/effects/adminhelp.ogg')
+			if(X.holder.rights == R_MENTOR)
+				to_chat(X, mentor_msg)// Mentors won't see coloring of names on people with special_roles (Antags, etc.)
+			else
+				to_chat(X, msg)*/
+	//show it to the person adminhelping too
+	to_chat(src, "<font color='blue'>PM to-<b>Staff</b> (<a href='?src=\ref[usr];close_ticket=\ref[ticket]'>CLOSE</a>): [original_msg]</font>")
+	var/admin_number_present = GLOB.admins.len - admin_number_afk
+	log_admin("HELP: [key_name(src)]: [original_msg] - heard by [admin_number_present] non-AFK admins.")
+	if(admin_number_present <= 0)
+		adminmsg2adminirc(src, null, "[html_decode(original_msg)] - !![admin_number_afk ? "All admins AFK ([admin_number_afk])" : "No admins online"]!!")
+	else
+		adminmsg2adminirc(src, null, "[html_decode(original_msg)]")
+
+	feedback_add_details("admin_verb","AH") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
+	return
+
+client/verb/bugreport()
+	set category = "Admin"
+	set name ="Submit Bug Report/Suggestions"
+	var url = "https://scp13.site/index.php?/forums/forum/8-bugs-and-suggestions/"
+	if(url)
+		if(alert("This will open the SCP13 Bug Report and Suggestions Forum Page in your Browser. Are you sure?",,"Yes","No")=="No")
+			return
+		src << link(url)
+
+client/verb/reportplayer()
+	set category = "Admin"
+	set name ="Report Player / Staff"
+	var url = "https://scp13.site/index.php?/forums/"
+	if(url)
+		if(alert("This will open the SCP13 Reports Forum Page in your Browser. This hasn't been made yet due to our Alpha stage. Are you sure?",,"Yes","No")=="No")
```

Comment:
We don't have a forum at all, remove this verb

Reply:
I forgot to remove this.